usaspending_api/common/logging.py

```diff
@@ -1,12 +1,30 @@
-from django.utils.timezone import now
-from django.utils.deprecation import MiddlewareMixin
-
+# Standard library imports
 import logging
+import os
 import sys
 import time  # time.perf_counter Matches response time browsers return more accurately than now()
 import traceback
 
-from typing import Optional, Callable
+# Django imports
+from django.utils.deprecation import MiddlewareMixin
+from django.utils.timezone import now
+
+# Typing imports
```

Comment:
The typing modules are part of the standard library so should go above.  No need to comment the section since pep 8 already lays out the pattern of imports: standard library -> third party -> local imports

Reply:
Done.